### DISCUSSION TITLE
Ability to change significant figures per indicator

### DISCUSSION THREAD

**Discussion by otis-bath** [open-sdg/open-sdg#1751] [Ideas]
Currently the significant figures used on the indicator pages are set in a site configuration option (custom.js) - however a topic expert has requested additional significant figures presented on an indicator. An option could be to make this an indicator configuration option so each one can be customised to overwrite the site settings.

**Comment by brockfanning** [comment]
One possible idea would be to pass a second parameter to the `opensdg.dataRounding` function, which contains some contextual information including the indicator number. If that is possible, it would allow each country to tailor the logic of the rounding per indicator.

**Reply by otis-bath** [reply]
@brockfanning That sounds promising - would we be able to test this out?
